packages/trackerless-network/src/logic/ContentDeliveryLayerNode.ts

```diff
@@ -390,6 +391,15 @@ export class ContentDeliveryLayerNode extends EventEmitter<Events> {
         return this.options.neighbors.getAll().map((n) => n.getPeerDescriptor())
     }
 
+    getInfo(): ContentDeliveryLayerNeighborInfo[] {
```

Comment:
```suggestion
    getInfos(): ContentDeliveryLayerNeighborInfo[] {
```

Reply:
This would be kind of grammatically wrong since there is no plural for information.

Reply:
True. Somehow we need to indicate that we return an array. How about e.g. `getInfoItems`? 

I think that `getInfos` would make the code easier to understand, and that the grammatical correctness doesn't improve readability here.

We've similarly used terms like `metadatas` when referring to arrays of stream metadata objects.

Reply:
It does. Also the method name should. When we read the code which uses this method, we see the methods name, not the return type
```
const foo = node.getInfos()
``` 

From this snippet we should see that the `foo` is an array. (Of course, we should have a good name for `foo`, preferrably matching the getter.)

Reply:
I don't really see this being necessary since the method is called getInfo in all other modules but whatever